Trie/Trie_add_Search_using_dict.py

```diff
@@ -0,0 +1,51 @@
+"""

```

Comment:
Just add a line here which explains the overall purpose of the code.

Reply:
Purpose:
* Implementation of Trie has been asked in my interviews like in Google 
* Store word in Dictionary 
* Search word in the dictionary returns True if exist
* Stores words in a compressed manner